backend/src/main/java/com/java/academy/gym/model/BaseEntity.java

```diff
@@ -0,0 +1,22 @@
+package com.java.academy.gym.model;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+
+@MappedSuperclass
+public class BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    protected Long id;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+}
```

Comment:
Also I would add here field which responsible for the optimistic lock something like:
@Column(name = "VERSION")
    @Version
    private Integer version;